boards/arm/bbc_microbit/bbc_microbit.dts

```diff
@@ -7,6 +7,32 @@
 /dts-v1/;
 #include <nordic/nrf51822_qfaa.dtsi>
 
+/* External edge connector pin mappings to nRF51 GPIO pin numbers.
```

Comment:
This was done because I wanted to get hold of the EXT_P0_GPIO_PIN define in DT board.overlay file. I do not expect a user to know that Microbit P0 connection is connected to P0.03 on the nRF51822 - and thus know that he should write ch0-pin = <3>; in his board.overlay file for the application. Using <EXT_P0_GPIO_PIN> made much more sense to me.

Reply:
Defines are removed again in new push - they were not used anywhere else so no need to change this file as part of this PR (if we do want to change it, that is.